Modelplace_Cloud_API/utils.py

```diff
@@ -0,0 +1,61 @@
+import requests
+import argparse
+import json
+import os
+
+
+def parse_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser(
+        description='Example of a Python application using the Modelplace Cloud API'
+    )
+    parser.add_argument(
+        "-e",
+        "--email",
+        type=str,
+        help="Set your email from your Modelplace.AI account.",
+        required=True,
+    )
+    parser.add_argument(
+        "-p",
+        "--password",
+        type=str,
+        help="Set your password from your Modelplace.AI account.",
+        required=True,
+    )
+    parser.add_argument(
+        "-m",
+        "--model-id",
+        type=int,
+        help="Model id. \n"
+            "To determine the model_id open model in Modelplace.AI "
+            "and copy the last number in the address bar: https://modelplace.ai/models/<model_id>",
+        required=True,
```

Comment:
e.g. for Tiny YOLO v4 (https://modelplace.ai/models/32), specify 32 as Model ID.